cwltool/schemas/draft-3/salad/schema_salad/validate.py

```diff
@@ -3,9 +3,11 @@
 import yaml
 import urlparse
 
+
```

Comment:
Whoops, the embedded copy shouldn't be updated like this; I will roll changes to `cwltool/schemas/*` back